1-exercises/A-this/exercise-3.js

```diff
@@ -8,8 +8,12 @@
 
 let bankAccount = {
   currentAmount: 100,
-  deposit: function (amount) {},
-  withdraw: function (amount) {},
+  deposit: function (amount) {
```

Comment:
Hi Dearest @HamraKhan ,
I'm doing the code review and have to pretend to know better than other's by leaving an unasked for line suggestion.
Instead of doing that I think I'll just leave tip, which you might already be familiar with.
I am a very lazy person and I love to use shorthands where ever possible.
And it just so happens that with method declarations that can be done very easily.
So if you ever feel lazy or just fancy a change, feel free to write this:
```
let bankAccount = {
    deposit: function (amount) { ... },
    withdraw: function (amount) { ... }
}
```
like this:
```
let bankAccount = {
    deposit (amount) { ... },
    withdraw (amount) { ... }
}
```
Thanks for listening to my TedTalk and have a lovely Valentine's Day! 💘